deploy/docker/scripts/supervisor-mongodb.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Let's enable xtrace so we get logs of what's happening when this script is run.

```suggestion
set -e
set -o xtrace
```

Reply:
i have updated this.